### DISCUSSION TITLE
How to uninstall .pkg installation?

### DISCUSSION THREAD

**Discussion by tillwestpfahl** [mittwald/cli#408] [Q&A] [answered]
I downloaded the latest arm64.pkg release and installed it, but I would like to switch to homebrew now. How do I uninstall the cli?

**Comment by martin-helmich** [comment, marked as answer]
Yeah, that's surprisingly difficult. 😕 

You _should_ (caveat: what follows is largely untested) be able to manually uninstall the package, by first removing all files returned by `pkgutil --files de.mittwald.cli` (in the installation directory returned by `pkgutil --pkg-info de.mittwald.cli`) and then "forgetting" the package by running `pkgutil --forget de.mittwald.cli`.[^1]

These pkg files suck. I'm beginning to wonder if we should just focus on Homebrew as the sole recommended way of installation, and just drop the pkgs altogether. 🤔 

[^1]: https://apple.stackexchange.com/a/371560

**Reply by tillwestpfahl** [reply]
That worked for me, thanks. I like the idea of not supporting pkg files altogether.
